sources/packages/web/src/components/common/EnvironmentBanner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="!isProduction" class="environment-banner">
+    <span class="environment-banner-text">{{ bannerText }}</span>
+  </div>
+</template>
+
+<script lang="ts">
+import { AppConfigService } from "@/services/AppConfigService";
+import { defineComponent, computed, ref, onMounted } from "vue";
+
+export default defineComponent({
+  setup() {
+    const appEnv = ref("");
+    const isProduction = ref(false);
```

Comment:
Minor. but do we require `isProduction` as well, as it is directly based on `appEnv`

Reply:
My point is, either `isProduction` can be computed property or the condition can  be directly used in v-if.  Either ways, the using the second ref could be avoided.